src/edit/DistortableCollection.Edit.js

```diff
@@ -204,20 +204,21 @@ L.DistortableCollection.Edit = L.Handler.extend({
 
   _addToolbar() {
     const group = this._group;
-    const map = group._map;
+    // const map = group._map;
 
     if (group.options.suppressToolbar || this.toolbar) { return; }
-
-    this.toolbar = L.distortableImage.controlBar({
-      actions: this.editActions,
-      position: 'topleft',
-    }).addTo(map, group);
+    // this is redundant as the controlbar is initialized in archive.js
```

Comment:
We could leave this as-is to avoid changing behaviors for other users of the library.